src/vs/workbench/contrib/positronNotebook/browser/PositronNotebookInstance.ts

```diff
@@ -623,6 +626,20 @@ export class PositronNotebookInstance extends Disposable implements IPositronNot
 				}
 			}, 0));
 		}
+
+		// After the content change fires and cells are synced, explicitly set the selection
+		// to maintain focus on the appropriate cell after deletion
+		if (targetFocusIndex !== null) {
+			this._register(disposableTimeout(() => {
```

Comment:
Is it possible to do this without a 0 timeout? We have a couple of these scattered around and I'm nervous it'll become hard to reason about down the line.